src/lib.rs

```diff
@@ -80,6 +80,95 @@ impl fmt::Display for Error {
 
 impl std::error::Error for Error {}
 
+/// Wrapper for native `Dav1dSettings` struct.
```

Comment:
```suggestion
/// Settings for creating a new [`Decoder`] instance.
```

Reply:
I'd like to point users towards native documentation, as it doesn't make sense for us to duplicate their docs here. Maybe something like:
```rust
/// Settings for creating a new [`Decoder`] instance.
/// See documentation for native `Dav1dSettings` struct.
```

Reply:
Makes sense, maybe even with a link to the documentation of the C library if there's any useful place to link to?

Reply:
I can't find a public documentation source pinned to a specific version of dav1d. [They do host current master docs](https://videolan.videolan.me/dav1d/structDav1dSettings.html), but as we discussed, that doesn't apply to the version of dav1d we're wrapping here. The C header seems to be the best source. I guess we could link to a tag in their git repo: https://code.videolan.org/videolan/dav1d/-/blob/0.9.2/include/dav1d/dav1d.h#L62 I'd personally just not include a link, up to you.

Reply:
Seems fine to me like this then, thanks :)